docs/devGuide/devGuide.md

```diff
@@ -175,12 +175,42 @@ On Windows:
   You should always check that the generated output is correct before committing any changes to the test sites.
 </box>
 
-##### New features
+#### Adding site content
 
 When adding new features, you should also add new site content into an existing test site or create a new test site to demonstrate the new feature. This is to ensure that your feature can be tested by building that test site.
 
+1. Add a new test page, e.g., `newTestPage.md`, containing a demonstration of the new feature.
+
+2. Open test\functional\test_site\site.json
```

Comment:
In the spirit of standardizing the paths in the document, we could change this to using forward slashes enclosed in fences -
```suggestion
2. Open `test/functional/test_site/site.json`
```

Reply:
Ah my bad. Thanks for noticing!